src/js/components/badge.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { Component, PropTypes } from 'react';
+
+import { openWidget } from '../services/app-service';
+import { SK_DARK_CONTRAST } from '../constants/styles';
+
+export class DefaultIcon extends Component {
```

Comment:
Yeah but it's exported and you said that export names should stand on their own :)